packages/react-components/react-search/src/components/SearchBox/useSearchBoxStyles.styles.ts

```diff
@@ -62,6 +54,9 @@ const useContentAfterStyles = makeStyles({
     paddingLeft: tokens.spacingHorizontalM,
     columnGap: tokens.spacingHorizontalXS,
   },
+  unfocused: {
+    display: 'none',
```

Comment:
Just thinking out loud here but it's not possible to animate `display: none` (for example, in the future we might want to fade this in/out).

Perhaps `opacity: 0;` (and `width/height: 0px` if we need it to not take up space) along with `aria-hidden=true` in the `useSearchBox` hook instead?

This should amount to the same behavior now but leave us option to adding `transition`s later (should we want to).

Details on why `opacity`: https://developer.chrome.com/blog/hardware-accelerated-animations/

Reply:
+1, though I'd actually skip `aria-hidden` -- my brain must've been elsewhere, but it's actually better for a11y to not hide the contentAfter when unfocused. That way touch screen readers and voice control can still consistently access the buttons